docs/features/hardware-offload/ovs-doca.md

```diff
@@ -0,0 +1,229 @@
+# OVS-DOCA
+
+## Introduction
+
+OVS supports Hardware Offload features which allows to offload OVS data-plane to the NIC while maintaining OVS control-plane unmodified. It is using SR-IOV technology with VF representor host net-device.
+
+DOCA-OVS extends traditional OVS-DPDK and OVS-Kernel data-path offload interfaces (DPIF), adds  OVS-DOCA as an additional DPIF implementation. It preserves the same interfaces as OVS-DPDK and OVS-Kernel while utilizing the DOCA Flow library. OVS-DOCA uses unique hardware offload mechanisms and application techniques to maximize performance and adds other features.
+
+## Motivation: Why use ovs-doca instead of ovs-dpdk
+
+OVS-DOCA is also a userland based virtual switch application like OVS-DPDK. Like OVS-DPDK it uses DPDK and PMD (Poll Mode Driver). It preserves the existing north bound APIs (Openflow) and is a drop in replacement for upstream OVS. But the main difference is that OVS-DOCA uses the DOCA-flow api instead of the rte_flow used by DPDK. This allows it to use hardware steering for offloads instead of software steering. The main benefits are:
```

Comment:
Is OVS-DOCA supported by OVS community like https://docs.openvswitch.org/en/latest/howto/dpdk/ ? Asking because all installation instructions are pointing to NVIDIA docs which might be a problem for u/s vendor neutral face lift.

Reply:
Currently it is not, but as far as I understand, we are working on upstreaming some of the software components. I will get back to you with a more precise answer or ask someone more informed to chime in here.

We expect OVS-DOCA to be used for its hardware offload capabilities which means it would be used with the  corresponding hardware as well. 

Since it provides mostly the same API to upper layers (OVN-Kubernetes etc.), only the installation and configuration instructions are referenced in the docs along with this document.